test_runner/src/main/kotlin/ftl/run/platform/android/CreateAndroidTestContext.kt

```diff
@@ -70,7 +70,9 @@ internal fun InstrumentationTestContext.getFlankTestMethods(
             .filter(testFilter.shouldRun)
             .filterNot(parameterizedClasses::belong)
             .map(TestMethod::toFlankTestMethod).toList()
-            .plus(parameterizedClasses.map(String::toFlankTestMethod))
+            .plus(parameterizedClasses
+                .filter { testFilter.shouldRun(TestMethod(it, emptyList())) }
+                .map(String::toFlankTestMethod))
```

Comment:
could we wrap
```kotlin
parameterizedClasses
                .filter { testFilter.shouldRun(TestMethod(it, emptyList())) }
                .map(String::toFlankTestMethod)
```
to some method ex:`parametrizedClassesTestMethodWhichShouldRun` 
to avoid nested chains, WDYT?